test/language/arguments-object/mapped/mapped-arguments-nonwritable-nonconfigurable-descriptors-4.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: Mapped arguments property descriptor change to non-writable and non-configurable
+description: >
+    Mapping stop working when property is set to non-writable. Change the
+    descriptor using [[DefineOwnProperty]] to {writable: false}, change argument[0]
+    value to 2 using [[DefineOwnProperty]] and then
+    change property descriptor to {configurable: false}.
+flags: [noStrict]
+esid: sec-arguments-exotic-objects-defineownproperty-p-desc
+includes: [propertyHelper.js]
+---*/
+
+function argumentsNonWritableThenNonConfigurableWithInterveningDefineOwnPropertyDescriptor(a) {
+  Object.defineProperty(arguments, "0", {writable: false});
+  Object.defineProperty(arguments, "0", {value: 2});
+  Object.defineProperty(arguments, "0", {configurable: false});
```

Comment:
Hum...I think this test case is different from the sample that you advised. In this case I want to verify if arguments[0].descriptor.value is updated even if {writable: false}. Doing Object.defineProperty(arguments, "0", {writable: false, enumerable: false}); makes it not viable.